application/package.rb

```diff
@@ -108,6 +110,9 @@ def main
                   if ['count', 'md5'].include?(configuration[:action])
                     status = "%s" % [result[:data][:output]]
                     puts(pattern % [result[:sender], status])
+                  elsif ['yum_checkupdates', 'apt_update', 'checkupdates', 'apt_checkupdates'].include?(configuration[:action])
+                    status = result[:data][:outdated_packages].map{ |package| package[:package] }.join(' ')
+                    puts(pattern % [result[:sender], status])
```

Comment:
why the 2 different formats? I think this is quite nice rather than the format above?

Reply:
I tough having more information is good, but perhaps this went a bit to far with returning all information. Updated code to return name + version in verbose mode